engine/include/cubos/engine/render/shadows/caster.hpp

```diff
@@ -0,0 +1,25 @@
+/// @file
+/// @brief Component @ref cubos::engine::ShadowCaster.
+/// @ingroup render-shadows-plugin
+
+#pragma once
+
+#include <glm/vec2.hpp>
+
+#include <cubos/core/reflection/reflect.hpp>
+
+#include <cubos/engine/api.hpp>
+
+namespace cubos::engine
+{
+    /// @brief Component which contains the options common to all shadow casters.
```

Comment:
```suggestion
    /// @brief Component which contains the settings common to all shadow casters.
```
Sounds better imo, but it's up to you.